gradle/libs.versions.toml

```diff
@@ -19,7 +19,7 @@ coroutines = "1.6.4"
 androidxNavigation = "2.5.3"
 appcompat = "1.4.1"
 dokka = "1.8.10"
-androidxCore = "1.8.0"
+androidxCore = "1.12.0"
```

Comment:
Didn't see anything concerning in these and allows us to use the new `ContextCompat.registerReceiver`